.github/workflows/workflow.yaml

```diff
@@ -1,6 +1,10 @@
 name: Go
 
-on: [push]
+on:
+  push:
+  pull_request:
+    branches:
+      - master
```

Comment:
why is that?

Reply:
It allows CI  checks to run on pull requests and not just when commuting to master. 

If I hadn’t done it, I wouldn’t have noticed my first implementation would have failed on versions Go 1.19 or earlier 😬